### ISSUE TITLE
Option to force square corners for all windows

### ISSUE THREAD

**Issue by joulev** [vinceliuice/WhiteSur-gtk-theme#416]
Currently all corners of non-maximised windows are rounded by default. However some apps are not following that well, and while the top corners work well, the bottom ones don't.

![image](https://user-images.githubusercontent.com/44609036/132123194-f5eb9089-d53f-492a-bb91-3bcc266c3fa0.png)

And there is that upstream Firefox bug that causes the corners to look glitchy...

Now we already have an option to force rounded corners even for maximised windows,

```
  --round, --roundedmaxwindow
 Set maximized window to rounded. Default is square
```

can we also have an option to force square corners for non-maximised windows?

**Comment by vinceliuice** [comment]
Only CSD gtk apps can set bottom round corner, those apps do not support that

**Comment by rivanfebrian123** [comment]
![Screenshot from 2021-09-05 19-32-54](https://user-images.githubusercontent.com/6300910/132126948-8b0a6075-49e8-4f36-9838-d00feeb84486.png)
![Screenshot from 2021-09-05 19-32-48](https://user-images.githubusercontent.com/6300910/132126949-c87d8a78-6755-4c58-8a9e-465a8b5efff4.png)

Styling GTK apps it self can be so tricky since GTK doesn't actually implement border radius clipping. That's why some widgets (for example, Matplotlib) cause a window to have "squared" corner(s). Notice how the bottom corner changes as I show and hide the sidebar.

**Comment by joulev** [comment]
Yeah, reading other issues here, I can understand getting rounded corners to work is simply impossible in many cases.

That’s why I’m not asking “why doesn’t it work” as a bug report, but I’m asking for a workaround as a feature request: option to disable any rounded corners in all cases.

> can we also have an option to force square corners for non-maximised windows?

**Comment by rivanfebrian123** [comment]
Oh, I got it. What do you think, @vinceliuice?

**Comment by vinceliuice** [comment]
I can add a option to disable the bottom rounded corners

**Comment by joulev** [comment]
> I can add a option to disable the bottom rounded corners

I mean the top corners too, since while I love beauty, I actually prefer consistency and different corner radius at the top and the bottom bothers my eyes quite much.

Other than that, the theme is beautiful enough for me :)
